src/commons/utils/DateHelper.ts

```diff
@@ -41,3 +41,9 @@ export const getStandardDate = (dateString: string): string => {
   const prettyDate = date.format('MMMM Do YYYY');
   return prettyDate;
 };
+
+export const getPrettyDateAfterHours = (dateString: string, hours: number): string => {
+  const date = moment(dateString).add(hours, 'hours');
+  const prettyDate = date.format('Do MMMM, HH:mm');
+  return prettyDate;
+};
```

Comment:
Would it be better if we use Moment's `.fromNow()` to get an even "prettier" format? E.g. "in 4 hours", "in 2 days"

Reply:
What do you think? Not a suggestion per se, but more to spark a discussion

Reply:
I think this is up to personal preference of the user. Personally, I would like to see the absolute deadline but I definitely understand that there wil be users who prefer a relative deadline. Maybe we can do both? e.g. " 22nd August, 21:00 (in 3 hours)"

Reply:
That's fair, let's try it out